### ISSUE TITLE
ModelRole.AddWindowsMember method fails when providing MemberID

### ISSUE THREAD

**Issue by alwortma** [TabularEditor/TabularEditor3#483]
TE3 v3.2.2

When running a C# script to add members to an existing role I get the following error:
`Property 'MemberID' of object 'ModelRoleMember' is immutable and can't be changed once set.`

I am following the syntax provided in TE2 Intellisense:
![image](https://user-images.githubusercontent.com/97995966/160007431-5066e89f-989b-4e66-ac42-029c30b577fa.png)

My code:
```
Model.Roles["Adminstrator"].AddWindowsMember("AMR\\sys_XXX","S-1-XXXXXXX-3684");
```

However, if I _only_ specify MemberName, then save/deploy the model, it seems as though the role member is properly recognized. Is this the intended functionality? If only the MemberName is required/allowed, then maybe the Intellisense prompt should be changed?

**Comment by otykier** [comment]
Yeah, this is a bug in the implementation of .AddWindowsMember. MemberId is indeed optional, but we will make sure that specifying it will also work.

**Comment by otykier** [comment]
This is now fixed in [3.2.3](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.2.3), where `MebmerId` can now be specified as an optional second argument.
